README.md

```diff
@@ -44,6 +44,8 @@ This appender uses [BigQueue](https://github.com/bulldog2011/bigqueue) implement
 | **bufferDir**          | *System.getProperty("java.io.tmpdir")* | Where the appender should store the buffer |
 | **socketTimeout**       | *10 * 1000*                                    | The socket timeout during log shipment |
 | **connectTimeout**       | *10 * 1000*                                    | The connection timeout during log shipment |
+| **addHostname**       | *false*                                    | Should we try and extract the hostname of the machine and add it as a field name "hostname". If we cant extract the hostname, we wont add anything. |
```

Comment:
"Optional. If true, the a field named 'hostname' will be added holding the host name of the machine. If from some reason there's no defined hostname, this field won't be added."